source/Playnite/Common/Extensions/StringExtensions.cs

```diff
@@ -86,6 +86,38 @@ public static string ToTileCase(this string source, CultureInfo culture = null)
             }
         }
 
+        public static bool IsStartOfStringAcronym(this string acronymStart, string input)
+        {
+            if (string.IsNullOrEmpty(input) || string.IsNullOrEmpty(acronymStart) || acronymStart.Length > input.Length)
+            {

```

Comment:
Also should probably return if acronymStart contains white space.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3202/commits/ca68783991e9974784db3a8909bbf8a0fc8b7288